<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daniel L. Contreras Pino</author>
 <class>wdgtNotas</class>
 <widget class="QWidget" name="wdgtNotas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>157</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>157</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">GEARC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconos/iconos/gearc.png</normaloff>:/iconos/iconos/gearc.png</iconset>
  </property>
  <layout class="QGridLayout" name="layoutcentral">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QScrollArea" name="frmNotas">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="wdgtNotasBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>432</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="lytNotas">
         <item>
          <spacer name="spcrNotas">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lnePromedio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="btnAgregarNota">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Agrega una nueva calificaci√≥n.</string>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnAutocalcular">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Autocalcula las notas restantes para alcanzar un promedio dado.</string>
     </property>
     <property name="text">
      <string>&amp;Autocalcular notas</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../recursos/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
